sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/PartitionPublishingProperties.cs

```diff
@@ -105,10 +105,10 @@ internal static PartitionPublishingProperties Empty
         /// <param name="ownerLevel">The owner level associated with the partition.</param>
         /// <param name="lastPublishedSequenceNumber">The sequence number assigned to the event that was last successfully published to the partition.</param>
         ///
-        protected internal PartitionPublishingProperties(bool isIdempotentPublishingEnabled,
-                                                         long? producerGroupId,
-                                                         short? ownerLevel,
-                                                         int? lastPublishedSequenceNumber) =>
+        internal PartitionPublishingProperties(bool isIdempotentPublishingEnabled,
```

Comment:
This is part of an unreleased feature; the model factory offers creation for it.  The protected modifier should not be applied here.